config/upcloud/group_vars/k8s_cluster/ck8s-k8s-cluster-upcloud.yaml

```diff
@@ -0,0 +1,30 @@
+etcd_deployment_type: kubeadm
+
+## Repo for UpClouds csi-driver: https://github.com/UpCloudLtd/upcloud-csi
+## To use UpClouds CSI plugin to provision volumes set this value to true
+## Remember to set UPCLOUD_USERNAME and UPCLOUD_PASSWORD
```

Comment:
So the CSI driver uses the same credentials as we use to authenticate with when provisioning the infrastructure using Terraform?

Reply:
We use UpCloud main account credentials to run kubespray which is also used by CSI driver.  I'll make it clearer in the docss